src/storage/gc_worker.rs

```diff
@@ -938,16 +933,14 @@ impl<S: GCSafePointProvider, R: RegionInfoProvider> GCManager<S, R> {
             *need_rewind = false;
             *end = None;
             info!(
-                "gc_worker: safe point updated to {}, auto gc will go to the end",
-                self.safe_point
+                "gc_worker: auto gc will go to the end"; "safe_point" => self.safe_point
             );
         } else {
             *need_rewind = true;
             *end = progress.clone();
             info!(
-                "gc_worker: safe point updated to {}, auto gc will rewind to {}",
-                self.safe_point,
-                end.as_ref().unwrap()
+                "gc_worker: auto gc will go to rewind"; "safe_point" => self.safe_point,
+                "next_rewind_key" => %(end.as_ref().unwrap())
```

Comment:
@MyonKeminta PTAL, is it ok to unwrap() here?

Reply:
`end`'s value is from `progress.clone()` and `progress` is unwrapped at L930, so here it must be ok to unwrap it.